### PR TITLE
Verify deploys

### LINKED ISSUE #416: Only allow specific users to push tags that trigger deploys
Similar to what is done in refine.bio

https://github.com/AlexsLemonade/refinebio/blob/dev/.circleci/verify_tag.sh

### PR DESCRIPTION
**Pull request by arielsvn**
## Issue Number

close #416 

## Purpose/Implementation Notes

Only allow authorized people to trigger deploys.

## Types of changes

* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Functional tests

We should be able to test this on staging.

### CODE REVIEW

File: .circleci/verify_tag.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script verifies that the tag triggering this deploy was signed
```
Comment (dongbohu) [inline .circleci/verify_tag.sh]: `deploy` should be `deployment`.

File: .circleci/verify_tag.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+# by a trusted member of the CCDL.
+
+for key in $(ls -1 keys/); do
```
Comment (dongbohu) [inline .circleci/verify_tag.sh]: This line can be simplified into:
```for key in keys/*; do```

File: .circleci/verify_tag.sh (hunk 1 above)
Comment (arielsvn) [reply .circleci/verify_tag.sh]:
```suggestion
# This script verifies that the tag triggering this deployment was signed
```

(trying github's suggestion feature)

File: .circleci/verify_tag.sh (hunk 2 above)
Comment (arielsvn) [reply .circleci/verify_tag.sh]:
```suggestion
for key in keys/*; do
```

**Comment by arielsvn** [comment]
I addressed the comments @dongbohu! and got to try a new github's feature. They now allow to make code suggestions in the comments

**Comment by dongbohu** [comment]
Interesting. Never heard of this feature!

**Comment by kurtwheeler** [comment]
:raised_hands: :raised_hands: :raised_hands:

**Pull request merged**